bcda-site-static/user_guide.md

```diff
@@ -62,9 +69,12 @@ Metadata about the Beneficiary Claims Data API is available as a FHIR [Capabilit
 `GET /api/v1/metadata`
 
 ###### cURL command
-```sh
+{%- capture code -%}
 curl https://sandbox.bcda.cms.gov/api/v1/metadata
-```
+{%- endcapture -%}
```

Comment:
can't remember what these tags do..

Reply:
captures the text inside as the variable "code"